### PR TITLE
fix: add Promise polyfill for IE11 🚑

### PR DESCRIPTION
**Pull request by gregorylegarec**

### CODE REVIEW

**Comment by gregorylegarec** [comment]
[Promise is undefined sous IE11](https://trello.com/c/dxF7kAP7/855-promise-is-undefined-sous-ie11)

File: assets/templates/login.html [hunk 1]
```diff
@@ -81,6 +81,7 @@ <h1>{{.Title}}</h1>
         </div>
       </main>
     </div>
+    <script src="{{asset .Domain "/scripts/polyfills/promise.js"}}"></script>
```
Comment (nono) [inline assets/templates/login.html]: Promise and Fetch needs polyfills only for IE? Maybe we can use a conditional comment to load the polyfills only on IE, like this:

```html
<!--[if IE]>
  <script src="{{asset .Domain "/scripts/polyfills/promise.js"}}"></script>
  <script src="{{asset .Domain "/scripts/polyfills/fetch.js"}}"></script>
<![endif]-->
```

Comment (nono) [reply assets/templates/login.html]: Forget that, it looks like IE11 doesn't support conditional comments...

Comment (gregorylegarec) [reply assets/templates/login.html]: Also @clochix has reported some user experimenting the same kind of problem on Android.

**Comment by gregorylegarec** [comment]
[Erreur de login sur IE](https://trello.com/c/AR2hp2n4/854-erreur-de-login-sur-ie)

**Pull request merged**